tgui/packages/tgui/interfaces/PreferencesMenu/QuirksPage.tsx

```diff
@@ -445,11 +446,15 @@ export function QuirksPage(props) {
             <Stack.Item basis="50%">
               <Stack vertical fill align="center">
                 <Stack.Item>
-                  <Box fontSize="1.3em">Quirk Balance</Box>
+                  <Box fontSize="1.3em">
+                    {pointsEnabled ? 'Quirk Balance' : 'Quirks Selected'}
+                  </Box>
                 </Stack.Item>
 
                 <Stack.Item>
-                  <StatDisplay>{balance}</StatDisplay>
+                  <StatDisplay>
+                    {pointsEnabled ? balance : selectedQuirks.length}
+                  </StatDisplay>
```

Comment:
Woudl rather you remove this entirely, the number of quirks you have is not useful with no limit

Reply:
I need to put something there to keep the webpage aligned... Removing it caused the whole list of selected quirks to move upwards. Any ideas?

Reply:
Put an empty div with some height

Reply:
Ok, I will attempt doing that!

Reply:
Fixed by commit [9bcd434](https://github.com/tgstation/tgstation/pull/81033/commits/9bcd4341d097e89ea709bd8bb10fe94cad472c31)